src/resolvers/projectResolver.ts

```diff
@@ -814,10 +793,10 @@ export class ProjectResolver {
       totalProjectUpdates: 1,
       verified: false,
       giveBacks: false,
+      adminUser: user,
     });
 
     const newProject = await this.projectRepository.save(project);
-    newProject.adminUser = await findUserById(Number(newProject.admin));
```

Comment:
No more virtual attribute, this is not required. Its already set on creation for adminUser in the createProject Mutation.